roles/mesos/defaults/main.yml

```diff
@@ -33,7 +33,7 @@ mesos_zk_acl_cmd: "docker run --rm -e ZK_AUTH=super:{{ zk_super_user_secret }} -
 
 # authentication options
 mesos_credentials: []
-mesos_authenticate_frameworks: "{{ mesos_credentials|length > 0 }}"
+mesos_authenticate_frameworks: false
```

Comment:
why was this changed? This just disabled framework authentication *for the whole project*... This needs to at least be set or unset through `security-setup`.